processor/servicegraphprocessor/processor_test.go

```diff
@@ -277,13 +277,18 @@ func verifyHappyCaseMetrics(t *testing.T, md pmetric.Metrics) {
 	assert.Equal(t, 1, sms.Len())
 
 	ms := sms.At(0).Metrics()
-	assert.Equal(t, 2, ms.Len())
+	assert.Equal(t, 3, ms.Len())
 
 	mCount := ms.At(0)
 	verifyCount(t, mCount)
 
-	mDuration := ms.At(1)
```

Comment:
Should this be kept for a couple of releases to allow people to upgrade to the new metric name? Which one would best represent the old one? Can you also add more info in the changelog telling folks what they should do not to have their dashboards broken?

Reply:
Added the legacy metric names under a feature flag and improved the description. Thanks for the review.